libcudacxx/include/cuda/std/detail/libcxx/include/version

```diff
@@ -201,234 +201,464 @@ __cpp_lib_void_t                                        201411L <type_traits>
+// However, if we are in __cuda_std__ we need to define our own symbols to not
+// conflict with the host stl.
+// At the same time we want bring in all feature test macros from host
+#ifdef __cuda_std__
+#if __has_include(<version>) // <version> should be the smallest include possible
+#include <version>
+#elif !defined(_LIBCUDACXX_COMPILER_NVRTC)
+#include <ciso646> // otherwise go for the smallest possible header
+#endif
+#endif // __cuda_std__
 
 #if defined(_LIBCUDACXX_USE_PRAGMA_GCC_SYSTEM_HEADER)
 #pragma GCC system_header
 #endif
 
+// We unconditionally define `__cccl_lib_meow` so that there is only one place to set the value
 #if _LIBCUDACXX_STD_VER > 11
-# define __cpp_lib_chrono_udls                          201304L
-# define __cpp_lib_complex_udls                         201309L
+# define __cccl_lib_chrono_udls                          201304L
+# define __cccl_lib_complex_udls                         201309L
 #ifdef _LIBCUDACXX_IS_CONSTANT_EVALUATED
-# define __cpp_lib_constexpr_complex                    201711L
+# define __cccl_lib_constexpr_complex                    201711L
 #endif
-# define __cpp_lib_concepts                             202002L
-# define __cpp_lib_exchange_function                    201304L
+# define __cccl_lib_concepts                             202002L
+# define __cccl_lib_exchange_function                    201304L
+// # define __cccl_lib_generic_associative_lookup           201304L
+# define __cccl_lib_integer_sequence                     201304L
+# define __cccl_lib_integral_constant_callable           201304L
+# define __cccl_lib_is_final                             201402L
+# define __cccl_lib_is_null_pointer                      201309L
+# define __cccl_lib_make_reverse_iterator                201402L
+// # define __cccl_lib_make_unique                          201304L
+# define __cccl_lib_null_iterators                       201304L
+// # define __cccl_lib_quoted_string_io                     201304L
+# define __cccl_lib_result_of_sfinae                     201210L
+# define __cccl_lib_robust_nonmodifying_seq_ops          201304L
+# if !defined(_LIBCUDACXX_HAS_NO_THREADS)
+// #   define __cccl_lib_shared_timed_mutex                 201402L
+# endif
+# define __cccl_lib_span                                 201902L
+// # define __cccl_lib_string_udls                          201304L
+# define __cccl_lib_transformation_trait_aliases         201304L
+# define __cccl_lib_transparent_operators                201510L
+# define __cccl_lib_tuple_element_t                      201402L
+# define __cccl_lib_tuples_by_type                       201304L
+#endif // _LIBCUDACXX_STD_VER > 11
+
+#if _LIBCUDACXX_STD_VER > 14
+# if defined(_LIBCUDACXX_ADDRESSOF)
+#   define __cccl_lib_addressof_constexpr                201603L
+# endif
+// # define __cccl_lib_allocator_traits_is_always_equal     201411L
+// # define __cccl_lib_any                                  201606L
```

Comment:
The following macro were not commented prior to your changes:

- `__cccl_lib_allocator_traits_is_always_equal`
- `__cccl_lib_any`
-  `__cpp_lib_string_udls` 
- `__cccl_lib_clamp`
- `__cccl_lib_enable_shared_from_this`
- `__cccl_lib_filesystem`
- `__cccl_lib_incomplete_container_elements`
- `__cccl_lib_map_try_emplace`
- `__cccl_lib_node_extract`
- `__cccl_lib_nonmember_container_access`
- `__cccl_lib_raw_memory_algorithms`
- `__cccl_lib_sample`
- `__cccl_lib_scoped_lock`
- `__cccl_lib_shared_mutex`
- `__cccl_lib_shared_ptr_weak_type`
- `__cccl_lib_string_view`

So the change doesn't look like rewording to me. If this is expected, I'd group these macro and add a comment that libcu++ is not supporting these for now.

Reply:
Those feature test macros have been plain wrong and should have never been defined by us

Reply:
Note that not defining them is the equivalent of us saying "this is not supported"